bee-common/bee-packable/src/packable/vec.rs

```diff
@@ -3,7 +3,7 @@
 
 extern crate alloc;
 
-use crate::{coerce::*, error::UnpackError, packer::Packer, unpacker::Unpacker, Packable};
+use crate::{error::UnpackError, packer::Packer, unpacker::Unpacker, Packable, UnpackErrorExt};
```

Comment:
I know this is a detail but here and in many places we import `UnpackError` from the `error` module and `UnpackErrorExt` from the root. Can we harmonise ?

Reply:
Oh yes, I'll remove those. Maybe we should make all those modules private?

Reply:
I kinda like importing through modules tbh... I like to know where things come from. If it was only me I don't event think I would have that much symbols at the root.

Reply:
would you mind if we let this be for now and discuss this in the packable meeting so I can open a PR with the decision we take?

Reply:
Yeah all good, I just feel like that's the kind of stuff we'll probably forget to change in the future.